src/patternfly/demos/Card/examples/Card.md

```diff
@@ -753,7 +753,9 @@ import './Card.css'
         {{/title}}
       {{/card-title}}
       {{#> card-actions}}
-        {{> dropdown dropdown--id=(concat card--id '-dropdown') dropdown-toggle--text="Filter" dropdown-menu--modifier="pf-m-align-right"}}
+        {{#> select id=(concat card--id '-select-dropdown')}}
```

Comment:
You can align these menus the same way as dropdowns - https://github.com/patternfly/patternfly/blob/a67b16945bec19cae3916d9688fe1e9c6cba2576/src/patternfly/components/Select/select.scss#L463-L465

Looks like it isn't in the select docs - could you add it to this section under `.pf-m-top`? https://pf4.patternfly.org/components/select#usage

Reply:
Yup, just found that. Fixed the docs.